### PR TITLE
Element geometry creation by brep operations.

### PR DESCRIPTION
**Pull request by bbastings**

### CODE REVIEW

File: common/api/imodeljs-backend.api.md [hunk 1]
```diff
@@ -431,8 +432,8 @@ export class BriefcaseDb extends IModelDb {
     readonly onBeforeClose: BeEvent<() => void>;
     readonly onChangesetApplied: BeEvent<() => void>;
     static readonly onCreated: BeEvent<(_imodelDb: BriefcaseDb) => void>;
-    static readonly onOpen: BeEvent<(_requestContext: AuthorizedClientRequestContext | ClientRequestContext, _briefcaseProps: BriefcaseProps) => void>;
-    static readonly onOpened: BeEvent<(_requestContext: AuthorizedClientRequestContext | ClientRequestContext, _imodelDb: BriefcaseDb) => void>;
+    static readonly onOpen: BeEvent<(_requestContext: ClientRequestContext | AuthorizedClientRequestContext, _briefcaseProps: BriefcaseProps) => void>;
```
Comment (pmconne) [inline common/api/imodeljs-backend.api.md:435]: You'll want to revert the change to these two lines. api-extractor has a bug.

File: core/common/src/geometry/ElementGeometry.ts [hunk 2]
```diff
@@ -136,6 +136,101 @@ export interface ElementGeometryUpdate {
...
+  /** The operation to perform on the input geometry */
+  operation: BRepGeometryOperation;
+  /** The input target and tool geometry (curves, surfaces, and solids) to perform the given operation on */
+  entryArray: ElementGeometryDataEntry[];
+  /** A function to call for the result geometry */
+  onResult: BRepGeometryFunction;
+  /** Whether disjoint results should be returned as multiple entries or a single disjoint body */
+  separateDisjoint?: boolean;
+  /** Options and parameters for this operation */
+  parameters?: any;
```
Comment (pmconne) [inline core/common/src/geometry/ElementGeometry.ts:216]: Expected parameters for operatoins documented anywhere?

Comment (bbastings) [reply core/common/src/geometry/ElementGeometry.ts:216]: Yes, there are interfaces like BRepHollowProps in same file.

**Review by pmconne — APPROVED** [review]
